src/Resources/contao/templates/backend/be_csv_import.html5

```diff
@@ -1,7 +1,18 @@
 <div id="tl_buttons">
     <a href="<?= ampersand($this->backUrl) ?>" class="header_back" title="<?= specialchars($GLOBALS['TL_LANG']['MSC']['backBTTitle']) ?>" accesskey="b"><?= $GLOBALS['TL_LANG']['MSC']['backBT'] ?></a>
 </div>
-<?= $this->message ?>
```

Comment:
Why are you not using `Message::generate()` here? It does effectively the same as your code.